charts/stable/notifiarr/templates/_initConf.tpl

```diff
@@ -0,0 +1,27 @@
+{{- define "notifiarr.init.conf" -}}
+{{- $uid := .Values.securityContext.container.runAsUser -}}
+{{- $gid := .Values.securityContext.container.runAsGroup -}}
+{{- $conf := (printf "%v/%v" .Values.persistence.config.targetSelector.main.main.mountPath "notifiarr.conf") -}}
+enabled: true
+# We run it on every boot just in case an upgrade (or the user)
+# manages to nuke the config file
+type: init
+imageSelector: alpineImage
+securityContext:
+  runAsUser: {{ $uid }}
+  runAsGroup: {{ $gid }}
```

Comment:
This is not needed, all containers (inlcuding init) inherit the above referenced values already